### PR TITLE
Add Compiling section to README.md

### PR DESCRIPTION
**Pull request by worldbeater**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ This project aims to be a full featured Telegram client with different approach
...
 
-To compile and run the application, you need to [download and install latest .NET Core SDK](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial). Then, clone the repository using [Git](https://git-scm.com/), go into `egram.tel/application/Tel.Egram` directory and run `dotnet run` command.
+To compile and run the application, you need to [download and install latest .NET Core SDK](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial). Clone the repository using [Git](https://git-scm.com/), you can use [Github Desktop](https://desktop.github.com/) if you are on Windows. Then, go into `egram.tel/application/Tel.Egram` directory and run `dotnet run` command.
 
 ```sh
 # Ubuntu 18.04 shell
-# Remember to install .NET Core SDK.
-apt install git
+# Remember to install .NET Core SDK before executing this.
+apt update && apt install -y git
```
Comment (x2bool) [inline README.md]: If we get rid of install instructions then next lines are going to be fully crossplatform. Besides that I don't want to teach people how to install software on their computers.

File: README.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ This project aims to be a full featured Telegram client with different approach
 
 ## Compiling
 
-To compile and run the application, you need to [download and install latest .NET Core SDK](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial). Then, clone the repository using [Git](https://git-scm.com/), go into `egram.tel/application/Tel.Egram` directory and run `dotnet run` command.
+To compile and run the application, you need to [download and install latest .NET Core SDK](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial). Clone the repository using [Git](https://git-scm.com/), you can use [Github Desktop](https://desktop.github.com/) if you are on Windows. Then, go into `egram.tel/application/Tel.Egram` directory and run `dotnet run` command.
```
Comment (x2bool) [inline README.md]: We are already on GitHub. I would just assume people know how to use git. No need for reference to external tools. Also build instructions are fully crossplatform, Windows-specific instructions are unnecessary.

File: README.md (hunk 1 above)
Comment (worldbeater) [reply README.md]: Yeah seems reasonable. Will change this.

**Comment by x2bool** [comment]
thank you

**Pull request merged**